src/mixins/planningActionMixin.js

```diff
@@ -166,8 +168,7 @@ export const planningActionMixin = {
       }
 
       if (event.type === INTERNAL_HOUR) {
-        const internalHour = this.internalHours.find(hour => hour._id === event.internalHour);
-        payload.internalHour = internalHour;
+        payload.internalHour = event.internalHour;
```

Comment:
c'est deja inclut dans cette ligne de code en fait : `let payload = { ...this.$_.omit(event, ['dates', '__v', '__index']) }`

Reply:
Done !

Reply:
je ne vois pas ton changement, la ligne est toujours la

Reply:
pardon c'est un oubli, corrigé !